packages/cna-template/template/nuxt/package.js

```diff
@@ -12,17 +12,25 @@ module.exports = {
 
     // Linter
     const eslint = linter.includes('eslint')
-    const lintStaged = eslint && linter.includes('lintStaged')
+    const lintStaged = linter.includes('lintStaged')
     const stylelint = linter.includes('stylelint')
     const prettier = linter.includes('prettier')
     const commitlint = linter.includes('commitlint')
     const lintScripts = {
       eslint: '<%= pmRun %> lint:js',
-      stylelint: '<%= pmRun %> lint:style'
+      stylelint: '<%= pmRun %> lint:style',
+      prettier: '<%= pmRun %> lint:prettier'
+    }
+    const lintfixScripts = {
+      eslint: "<%= pmRun %> lint:js <%= pm === 'npm' ? '-- ' : '' %>--fix",
```

Comment:
I used EJS templates because there already was `<%= pmRun %>`, but I don't see a reason not to `` `use ${variables} instead` ``; especially as any incorrect templates in this JS cause a compilation error, e.g. if you escape `\'` within `"` instead of replacing `'` by `"` and using `'` inside. Also the whitespace-trim didn't seem to be very useful here.